### PR TITLE
Overridden CSS Property removed - Fixes #144

### LINKED ISSUE #144: 1 CSS property is overridden
## Description
1 CSS property is overridden

Generally, it is unnecessary to define 2 times the same property within the same rule: the second one overrides the first.
This may affect the readability of CSS code. Eliminate unnecessarily overridden properties also reduce the file size.

How to improve it?
Remove one occurrence of the duplicated property. For example, the following properties:

## Mocks
[INSERT RELEVANT PNG FILE]
.myClass {
  margin: 20px;
  ...
  margin: 10px;
}
Should be replaced by:

.myClass {
  margin: 10px;
}
main-menu:target + .bac[...]ckdrop {background: rgba(0,0,0,.85)} (line 129, col 3)
## Acceptance Criteria
styles.css should be modified accordingly.
### Update [Required]
- [ ] styles.css

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
5-10 min

### PR DESCRIPTION
**Pull request by Vertika-Shukla**
Overridden "background" property from "main-menu:target + .backdrop" is removed

### Description
Overridden "background" property from "main-menu:target + .backdrop" is removed
.main-menu:target + .backdrop{ background: rgba(0,0,0,.85); }

Fixes #144 

### Type of Change
- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Changes made successfully run on my local machine.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+      -moz-box-shadow: none;
+            box-shadow: none;
+    height: auto;
+    width: auto;
+    background: none;
+  }
+
+  .main-menu a {
+    color: rgb(98,181,229);
+    border: 0 !important; /* Remove borders from off-canvas styling */
```
[the remark this answers is not in the document]
Comment (AnkitKhatri55555) [reply styles.css]: Can you mention the lines at which you made changes.Here all lines are shown as added

[the remark this answers is not in the document]
Comment (Vertika-Shukla) [reply styles.css]: @AnkitKhatri55555 Line 128 which was previously "background: #000" is removed and replaced by line 129 that is "background: rgba(0,0,0,.85"

**Comment by divyanshu-rawat** [comment]
@Tharangi let's close this.

**Comment by Tharangi** [comment]
Closing this.

**Pull request closed without merging**